frontend/device/android/device.lua

```diff
@@ -254,12 +250,19 @@ function Device:performHapticFeedback(type)
 end
 
 function Device:retrieveNetworkInfo()
-    local ssid, ip, gw = android.getNetworkInfo()
-    if ip == "0" or gw == "0" then
+    local ok, type = android.getNetworkInfo()
+    if ok == "0" then
         return _("Not connected")
     else
-        local BD = require("ui/bidi")
-        return T(_("Connected to %1\n IP address: %2\n gateway: %3"), BD.wrap(ssid), BD.ltr(ip), BD.ltr(gw))
+        local status = nil
+        if type == "1" then
+            status = _("connected to Wi-Fi")
```

Comment:
Unless you want to change all the strings, it seems better to keep them consistent? :-)
```suggestion
            status = _("Connected to Wi-Fi")
```